probe-rs-tools/src/bin/probe-rs/cmd/info.rs

```diff
@@ -294,11 +295,11 @@ fn show_arm_info(interface: &mut dyn ArmProbeInterface, dp: DpAddress) -> Result
     let num_access_ports = interface.num_access_ports(dp)?;
 
     for ap_index in 0..num_access_ports {
-        let ap = ApAddress {
-            ap: ap_index as u8,
+        let ap = FullyQualifiedApAddress {
+            ap: probe_rs::architecture::arm::ApAddress::V1(ap_index as u8),
```

Comment:
Please import this enum in all places. Alternatively, `FullyQualifiedApAddress::new_v1` would read much better.

Reply:
In a further PR (still incomplete) I’ll introduce those `new_v1` at the `FullyQualifiedApAddress` with eg `FullyQualifiedApAddress::new_v1_with_dp()` etc.

Reply:
Hopefully the latest changes resolve this :)